src/example/p4p5/profiles/uninstall-plone4/cssregistry.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
@mauritsvanrees tnx for explanation. @ebrehault @hvelarde the removal of those unregistry profile files should be reverted, IMO. explicit is better than implicit.